huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go

```diff
@@ -357,7 +571,7 @@ locals {
 
 resource "huaweicloud_dms_rocketmq_instance" "test" {
   name              = "%[2]s"
-  engine_version    = element(local.query_results.versions, length(local.query_results.versions)-1)
+  engine_version    = local.query_results.versions[1]
```

Comment:
if you use index 1, that means the list has at least 2 elements.
are you sure about that?

Reply:
done